engine/src/main/java/io/camunda/zeebe/engine/processing/EngineProcessors.java

```diff
@@ -97,7 +98,12 @@ public static TypedRecordProcessors createEngineProcessors(
         deploymentDistributionCommandSender,
         zeebeState.getKeyGenerator());
     addMessageProcessors(
-        bpmnBehaviors, subscriptionCommandSender, zeebeState, typedRecordProcessors, writers);
+        bpmnBehaviors,
+        subscriptionCommandSender,
+        zeebeState,
```

Comment:
❌ If we choose to change `zeebeState` to `processingState` then we should do this consistently. Otherwise, this will lead to confusion.

Reply:
Do you like the renaming in general? If so, I can make the changes and split them off into a separate commit.

Reply:
💭 TBH, I have to get used to it a bit. I do not see better names now, and `processingState` is quite fitting.

👍 I'm happy to see it change to `processingState` everywhere. 

💭 Like I mentioned, I could also take this off your hands, as it's getting you deeper into engine territory 😄 I'm also happy to just review it.

Reply:
Done, all renamings are in 9d46d3d